CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.5.2
+
+* Add `full-width` flag to remove left and right padding when using a full width page header. 
```

Comment:
Can you put the PR info for that change here too?

Reply:
Damn! Cheers for picking that up!